content/graphql/resolvers-map.md

```diff
@@ -350,9 +426,9 @@ export class CreatePostInput extends Post {
 }
 ```
 
-#### Decorators
+#### GraphQL resolver argument decorators
```

Comment:
Maybe just `Arguments decorators` or `GraphQL argument decorators`? Since they can be used for subscriptions, queries, mutations, field resolvers etc